container-deployment/install-by-docker.md

```diff
@@ -19,7 +19,8 @@ Then, download Fluentd edge-debian's \(edge-debian means latest version of Fluen
 $ docker pull fluent/fluentd:edge-debian
 ```
 
-Debian and Alpine Linux version is available for Fluentd image. Debian version is recommended officially since it has [`jemalloc`](https://github.com/jemalloc/jemalloc) support. However, the Alpine image is smaller.
+Debian and Alpine Linux versions are available for Fluentd image. Debian version is recommended officially since it has [`jemalloc`](https://github.com/jemalloc/jemalloc) support. However, the Alpine image is smaller.
+And, Windows server version also is available.
```

Comment:
Thanks for this update!

It looks like this is more natural to me.

```suggestion
And, Windows server version is also available.
```

Reply:
Thanks. Fixed.